src/hooks/Trades.ts

```diff
@@ -17,35 +18,24 @@ function useAllCommonPairs(currencyA?: Currency, currencyB?: Currency): Pair[] {
     ? [wrappedCurrency(currencyA, chainId), wrappedCurrency(currencyB, chainId)]
     : [undefined, undefined]
 
+  const customBases = CUSTOM_BASES[chainId]
+  const customBasesA: Token[] = customBases && tokenA ? customBases[tokenA.address] : []
+  const customBasesB: Token[] = customBases && tokenB ? customBases[tokenB.address] : []
+  const basesWithCustom: Token[] = union(bases, customBasesA, customBasesB)
```

Comment:
let's try to get rid of the dependency if we can. something like:

`[...bases, ...customBasesA, ...customBasesB].filter((base, i, allBases) => allBases.findIndex(allBase => allBase.equals(base)) === i)`

should work